packages/react/src/TextInput/TextInput.tsx

```diff
@@ -41,17 +41,7 @@ export type TextInputNonPassthroughProps = {
   trailingAction?: React.ReactElement<React.HTMLProps<HTMLButtonElement>>
 } & Pick<
   StyledWrapperProps,
-  | 'block'
-  | 'contrast'
-  | 'disabled'
-  | 'monospace'
-  | 'sx'
-  | 'width'
-  | 'maxWidth'
-  | 'minWidth'
```

Comment:
Removing props like this is typically associated with a deprecate -> remove in next major release flow instead of being included in a single minor release. Is this something that we could batch up for the [next major release](https://github.com/github/primer/issues/3541) and deprecate in the current minor?

Reply:
Updated and added to the major release issue: https://github.com/github/primer/issues/3541#issuecomment-2474329371